src/authentication/team-token.ts

```diff
@@ -0,0 +1,58 @@
+import * as Knex from 'knex';
+import * as moment from 'moment';
+
+const randomstring = require('randomstring');
+
+export const teamTokenLength = 7;
+
+export interface TeamToken {
+  teamId: number;
+  token: string;
+  createdAt: number | moment.Moment;
+}
+export function teamTokenQuery(db: Knex, token?: string, teamId?: number) {
```

Comment:
What is this function's purpose? It generates a query and allows you to pass a token and/or a teamId? What happens if you only pass one, or decide to pass both?